ios/Classes/SwiftAlarmPlugin.swift

```diff
@@ -55,7 +55,7 @@ public class SwiftAlarmPlugin: NSObject, FlutterPlugin {
             if id == nil {
```

Comment:
I made a mistake here. `id` is not equal to `call.arguments`. It has to be extracted from a map, like in `stopAlarm` above:

```Swift
if let arguments = call.arguments as? [String: Any],
    let id = arguments["id"] as? Int {
        // id is specified
} else {
        // id is not specified
}
```

Reply:
I just released the hotfix in `4.0.11`.

Reply:
Thanks!